tests/org.palladiosimulator.dataflow.confidentiality.analysis.tests/src/org/palladiosimulator/dataflow/confidentiality/analysis/AnalysisFeatureTest.java

```diff
@@ -0,0 +1,53 @@
+package org.palladiosimulator.dataflow.confidentiality.analysis;
+
+import static org.palladiosimulator.dataflow.confidentiality.analysis.AnalysisUtils.TEST_MODEL_PROJECT_NAME;
+
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.palladiosimulator.dataflow.confidentiality.analysis.testmodels.Activator;
+
+public class AnalysisFeatureTest {
+    protected static StandalonePCMDataFlowConfidentialtyAnalysis onlineShopAnalysis;
```

Comment:
Have these to be static?

Reply:
With the `@TestInstance(Lifecycle.PER_CLASS)` in the Base Test, i got the test to work without the static attributes on the example analysis